### DISCUSSION TITLE
about_site/%E5%85%B3%E4%BA%8E%E6%9C%AC%E7%AB%99

### DISCUSSION THREAD

**Discussion by giscus** [erduotong/erduotong.github.io#1] [Announcements]
# about_site/%E5%85%B3%E4%BA%8E%E6%9C%AC%E7%AB%99

这里是关于本站

https://erduotong.github.io/about_site/%E5%85%B3%E4%BA%8E%E6%9C%AC%E7%AB%99.html

**Comment by erduotong** [comment]
看起来，这是**第一个评论**呢！
